ios/MaasNearbyTransit/Sources/MaasNearbyTransit/Compositions/NearbyTransitView.swift

```diff
@@ -0,0 +1,60 @@
+import SwiftUI
+
+public struct NearbyTransitView: View {
+
+    let iitems = [
+        NearbyTransitComponentsPreviewData.stopWithSchedulesWithDepartures,
+        NearbyTransitComponentsPreviewData.stopWithSchedulesWithDepartures,
+        NearbyTransitComponentsPreviewData.stopWithSchedulesWithDepartures,
+        NearbyTransitComponentsPreviewData.stopWithSchedulesWithDepartures,
+        NearbyTransitComponentsPreviewData.stopWithSchedulesWithDepartures,
+        NearbyTransitComponentsPreviewData.stopWithSchedulesWithDepartures,
+        NearbyTransitComponentsPreviewData.stopWithSchedulesWithDepartures,
+        NearbyTransitComponentsPreviewData.stopWithSchedulesWithDepartures,
+    ]
```

Comment:
this should be somewhere under #if DEBUG and passed via init probably

Reply:
Yup, 👍 this now will be inside preview content and whole NearbyTransitView will only have a constructor with required input data.